site/src/pages/docs/advanced-structured-logging.mdx

```diff
@@ -283,7 +283,7 @@ import com.linecorp.armeria.common.logging.ContentPreviewerFactoryBuilder;
 ServerBuilder sb = Server.builder();
 
 ContentPreviewerFactoryBuilder builder = ContentPreviewerFactory.builder().maxLength(100);
-builder.text(StandardCharsets.UTF_8 /* default charset */, (ctx, headers) -> {
+builder.text((ctx, headers) -> {
```

Comment:
I guess the intention of this was
`builder.defaultCharset(StandardCharsets.UTF_8).text(...)`. 😄 
But yeah, I'm fine with this. Thanks!

Reply:
Yeah, I thought about whether put `defaultCharset(StandardCharsets.UTF_8)` or not.

Thank you!